cli.Dockerfile

```diff
@@ -1,14 +1,34 @@
+# docker build -t robusta-cli -f cli.Dockerfile .
```

Comment:
Please add a comment explaining this better. E.g. "Test test this locally, run..."